covsirphy/cleaning/linelist.py

```diff
@@ -199,13 +199,11 @@ def subset(self, country, province=None):
         df = df.drop([self.COUNTRY, self.PROVINCE], axis=1)
         return df.reset_index(drop=True)
 
-    def closed(self, country, province=None, outcome="Recovered"):
+    def closed(self, outcome="Recovered"):
```

Comment:
Tests with pytest and Semaphore CI failed.

Please update test codes in tests/test_dataloader.py::TestLineListData::test_closed and example codes in example/load_linlist.py.

The results of tests is shown at the bottom of conversation tab of this pull request and https://semaphoreci.com/lisphilar/covid19-sir

Reply:
Please add a comment on #362 because `country` and `province` arguments were removed.

Reply:
Oh sorry, I had not considered these dependencies for the example and test codes. I will try to fix this soon.